data/sql/updates/pending_db_world/rev_1609100844918098600.sql

```diff
@@ -0,0 +1,131 @@
+(@PATH, 1, 2277.19, 5177.20, 11.33, 0, 0, 0, 0, 100, 0),
+(@PATH, 2, 2268.98, 5175.54, 11.166, 0, 0, 0, 0, 100, 0),
+(@PATH, 3, 2254.60, 5188.90, 11.385, 0, 0, 0, 0, 100, 0),
+(@PATH, 4, 2256.71, 5195.323, 11.450, 0, 0, 0, 0, 100, 0),
+(@PATH, 5, 2260.67, 5200.36, 11.3711, 0, 0, 0, 0, 100, 0),
+(@PATH, 6, 2264.38, 5199.09, 11.3676, 0, 13000, 0, 0, 100, 0),
+(@PATH, 7, 2270.18, 5196.94, 12.4959, 0, 0, 0, 0, 100, 0),
+(@PATH, 8, 2275.23, 5194.94, 12.49, 0, 0, 0, 0, 100, 0),
+(@PATH, 9, 2276.23, 5197.58, 12.4896, 0, 0, 0, 0, 100, 0),
+(@PATH, 10, 2279.08, 5206.5, 12.494, 0, 60000, 0, 0, 100, 0),
+(@PATH, 11, 2278.6, 5205.3, 12.4926, 0, 16000, 0, 0, 100, 0),
+(@PATH, 12, 2275.55, 5197.84, 12.4897, 0, 0, 0, 0, 100, 0),
+(@PATH, 13, 2273.063, 5196.067, 12.4932, 0, 0, 0, 0, 100, 0),
+(@PATH, 14, 2265.21, 5198.54, 11.3533, 0, 0, 0, 0, 100, 0),
+(@PATH, 15, 2260.26, 5202.83, 11.4377, 0, 0, 0, 0, 100, 0),
+(@PATH, 16, 2258.98, 5205.83, 11.322, 0, 0, 0, 0, 100, 0),
+(@PATH, 17, 2268.17, 5226.07, 11.2421, 0, 5000, 0, 0, 100, 0);
+
+DELETE FROM `creature_formations` WHERE `leaderGUID`= 85221;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES 
+(85221, 85221, 0, 0, 4, 0, 0),
+(85221, 85226, 1, 305, 4, 8, 4),
+(85221, 85222, 1, 55, 4, 8, 4);
+                                                                                                                  
+DELETE FROM `creature_text` WHERE `CreatureID` IN (25245, 25258, 25259, 25260, 25261);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(25245, 0, 0, 'You lads know the drill.  No alcohol to servicemen on duty.  You want the general to kill me himself?', 12, 7, 100, 274, 0, 0, 24744, 0, 'James Deacon'),
+(25245, 1, 0, 'All right, all right... but you didn\'t get these from me!', 12, 7, 100, 6, 0, 0, 24745, 0, 'James Deacon'),
+(25258, 0, 0, 'One last drink before we head to the front?', 12, 7, 100, 1, 0, 0, 24475, 0, 'Footman Rob'),
+(25258, 1, 0, 'Barkeep!  Bring out your strongest ale.  We\'re off to the front.', 12, 7, 100, 1, 0, 0, 24478, 0, 'Footman Rob'),
+(25258, 2, 0, 'You\'ll surely make an exception for us, eh?  We might not come back.', 12, 7, 100, 1, 0, 0, 24479, 0, 'Footman Rob'),
+(25258, 3, 0, 'I guess that\'s it then.  Off we go.', 12, 7, 100, 1, 0, 0, 24482, 0, 'Footman Rob'),
+(25259, 0, 0, 'Sure, why not?', 12, 7, 100, 6, 0, 0, 24476, 0, 'Footman George'),
+(25260, 0, 0, 'Hey, guys!  Wait up!', 12, 7, 100, 0, 0, 0, 24761, 0, 'Footman Mitch'),
+(25261, 0, 0, 'That was the worst beer I\'ve ever had.', 12, 7, 100, 0, 0, 0, 24483, 0, 'Footman Chuck');
+                                                                                                                  
```

Comment:
It is more for a guidance. But it is triggering because you have two newlines instead of one

Reply:
Good catch, hope it will work correctly now.

Reply:
You still have two newlines, should only be one.

Reply:
Then it is a bug from github, i will edit this later from my pc and push to git after

Reply:
Fixed now.